### PR TITLE
adding helm chart

### LINKED ISSUE #50: Helm chart
#### Request / Offer

This tool looks great for us. Is there a helm chart (maybe in a private repo?). If so can it be made public, and if not would you like one?

### PR DESCRIPTION
**Pull request by ssandy83**
Adding helm chart.
fixes #50

### CODE REVIEW

File: examples/helm-chart/governor/templates/node-reaper.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            # Expose self nodeName as NODE_NAME environment variable - required for reapOld
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            image: governor:latest
```
Comment (eytan-avisror) [inline examples/helm-chart/governor/templates/node-reaper.yaml:38]: You may want to template the image version vs. using latest ?

**Comment by eytan-avisror** [comment]
Thanks @ssandy83 
Had a comment regarding image version.. I am assuming you tested this works?

**Comment by ssandy83** [comment]
> Thanks @ssandy83
> Had a comment regarding image version.. I am assuming you tested this works?

Thanks for the review, @eytan-avisror 
With the new commit, I have templatized the image version.
While testing, I found that the pod-reaper was erroring out with no permission to list namespaces. To fix that, I have added namespaces in the allowed resources for pod-reaper cluster role. let me know if this is fine or we need to be more restrictive.

**Comment by eytan-avisror** [comment]
Thanks @ssandy83 
We recently added namespace listing to pod reaper and I guess we forgot to update the RBAC in our example.
I think it would be more accurate if you separate pods and namespaces to this:
```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: pod-reaper
rules:
- apiGroups: [""]
  resources: ["pods"]
  verbs: ["get", "delete", "list"]
- apiGroups: [""]
  resources: ["namespaces" ]
  verbs: ["list"]
```

This way we only add permission to `list` namespaces.
Do you mind also updating this here:
https://github.com/keikoproj/governor/blob/master/examples/pod-reaper.yaml#L12-L20

**Comment by ssandy83** [comment]
> Thanks @ssandy83
> We recently added namespace listing to pod reaper and I guess we forgot to update the RBAC in our example.
> I think it would be more accurate if you separate pods and namespaces to this:
> 
> ```
> kind: ClusterRole
> apiVersion: rbac.authorization.k8s.io/v1
> metadata:
>   name: pod-reaper
> rules:
> - apiGroups: [""]
>   resources: ["pods"]
>   verbs: ["get", "delete", "list"]
> - apiGroups: [""]
>   resources: ["namespaces"]
>   verbs: ["list"]
> ```
> 
> This way we only add permission to `list` namespaces.
> Do you mind also updating this here:
> https://github.com/keikoproj/governor/blob/master/examples/pod-reaper.yaml#L12-L20

Thank you, @eytan-avisror 
I have made changes as suggested.

**Review by eytan-avisror — APPROVED** [review]
Thanks!

**Pull request merged**